packages/cdktf-cli/src/bin/cmds/helper/performance.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import pidtree from "pidtree";
+import pidusage from "pidusage";
+
+async function getCombinedMemoryUsageInBytes(
```

Comment:
A comment here for the future would be nice. I assume we're doing this because JSII pacmak does this multi-process invocation for doing the conversion, right?